src/Behat/Testwork/EventDispatcher/TestworkEventDispatcherPhp72Trait.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+/**
+ * This trait is created to allow us to have PHP7.2 code in this project and still
+ * be able to support older PHP versions.
+ *
+ * @author Tobias Nyholm <tobias.nyholm@gmail.com>
+ */
+trait TestworkEventDispatcherPhp72Trait
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function dispatch($event, string $eventName = null): object
```

Comment:
```suggestion
    public function dispatch($event, ?string $eventName = null): object
```

Let's be strict :}

Reply:
Thank you @Taluu. I let @japicoder fix this on the Behat PR if he wants to.

I've never understood this. 
With my code: 
```php
$dispatcher->dispatch($object);
$dispatcher->dispatch($object, 'foo');
```

With your suggestion: 
```php
$dispatcher->dispatch($object);
$dispatcher->dispatch($object, 'foo');
$dispatcher->dispatch($object, null);
```

-------

Why would I want to allow passing null? I see some default rules in php-cs-fixer suggest the same.

Reply:
It's just because of your `string $eventName = null`, which allows null but is kinda a hack from pre-7.1 : if you want a nullable (something), then you should prepend a `?` on the type.

For property types in 7.4, you can't have `private string $foo = null` : you must prepend a `?` on the type (`private ?string $foo = null`).

With your code, it's still possible to pass `$dispatcher->dispatch($object, null);`, it's actually the same meaning (kinda actually). And I guess it's in the contract, so...

Reply:
Hm. okey. I did not know. Thank you